docs/modules/ROOT/pages/index.adoc

```diff
@@ -0,0 +1,21 @@
+= The jekyll-asciidoc Jekyll Site Generator Extension
```

Comment:
Can you simplify this title to "AsciiDoc Plugin for Jekyll" or "AsciiDoc Support for Jekyll" or "Jekyll AsciiDoc Integration"? The current title is too long.